website/screens/principles/color/ColorPage.tsx

```diff
@@ -0,0 +1,772 @@
+                </td>
+                <td>#947705</td>
+                <td>(48, 93%, 30%)</td>
+              </tr>
+              <tr>
+                <td>
+                  <Code>color-yellow-900</Code>
+                </td>
+                <td>#624f04</td>
+                <td>(48, 93%, 20%)</td>
+              </tr>
+            </tbody>
+          </DxcTable>
+          <HeadingLink level={4}>Orange</HeadingLink>
+          <DxcTable>
+            <thead>
+              <tr>
+                <th>Name</th>
+                <th>Value (hex)</th>
+                <th>Value (hsl)</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td>
+                  <Code>color-orange-50</Code>
+                </td>
+                <td>#fefaf5</td>
+                <td>(32, 90%, 98%)</td>
+              </tr>
+              <tr>
+                <td>
+                  <Code>color-orange-100</Code>
+                </td>
+                <td>#fef3e7</td>
+                <td>(32, 90%, 95%)</td>
+              </tr>
+              <tr>
+                <td>
+                  <Code>color-orange-200</Code>
+                </td>
+                <td>#fce7cf</td>
+                <td>(32, 90%, 90%)</td>
+              </tr>
+              <tr>
+                <td>
+                  <Code>color-orange-300</Code>
+                </td>
+                <td>#facf9e</td>
+                <td>(32, 90%, 80%)</td>
+              </tr>
+              <tr>
+                <td>
+                  <Code>color-orange-400</Code>
+                </td>
+                <td>#f7b76e</td>
+                <td>(32, 90%, 70%)</td>
+              </tr>
+              <tr>
+                <td>
+                  <Code>color-orange-500</Code>
+                </td>
+                <td>#f59f3d</td>
+                <td>(32, 90%, 60%)</td>
+              </tr>
+              <tr>
+                <td>
+                  <Code>color-orange-600</Code>
+                </td>
+                <td>#f38f20</td>
+                <td>(32, 90%, 54%)</td>
+              </tr>
+              <tr>
+                <td>
+                  <Code>color-orange-700</Code>
+                </td>
+                <td>#c26c0a</td>
+                <td>(32, 90%, 40%)</td>
+              </tr>
+              <tr>
+                <td>
+                  <Code>color-orange-800</Code>
+                </td>
+                <td>#915108</td>
+                <td>(32, 90%, 30%)</td>
+              </tr>
+              <tr>
+                <td>
+                  <Code>color-orange-900</Code>
+                </td>
+                <td>#613605</td>
+                <td>(32, 90%, 20%)</td>
+              </tr>
+            </tbody>
+          </DxcTable>
+        </DxcStack>
+        <DocFooter githubLink="https://github.com/dxc-technology/halstack-style-guide/blob/master/guidelines/principles/colors/README.md" />
```

Comment:
Shouldn't the URL point to the file itself? `website/screens/principles/color/ColorPage.tsx` instead of `guidelines/principles`?